### PR TITLE
feat(rfc): unity swagger.yml Users and Invites draft

### PR DESCRIPTION
**Pull request by 121watts**
Closes: #287 

This is a draft for the `Users` and `Invites` unification API client.  If we can knock out a contract between Quartz and the `ui` then e-commerce and ui can start working in parallel 😸 .

   - [x] Delete User
   - [x] Get Users
   - [x] Delete Invitation
   - [x] Resend Invitation
   - [x] Create Invitation

### CODE REVIEW

**Comment by subirjolly** [comment]
I will give it a 👍 instead of approving. Don't consider myself qualified to approve, yet.

**Review by randycoulman — APPROVED** [review]
Overall, this looks about right, though I haven't had time to do a detailed comparison against what's actually implemented in Quartz right now, so we might need a couple of minor changes later.

This looks like what used to live in the influxdb swagger.  It looks like most of that was removed by https://github.com/influxdata/influxdb/pull/19102, but there's still some cruft left over there that we might want to remove.  I also noticed that the above-mentioned PR removed the `/orgs/{orgID}/owners/{userID}` endpoint, but it hasn't been added back in this PR.  However, I think that might be an error in the previous PR because I think that's still a valid endpoint in the Influx API.

My only other thought is that I think this Swagger spec should ultimately live in the Quartz repo because that's where it will be implemented, but I realize that makes it harder to generate code from it. If it stays here, I'd want to be sure that we're OK with publishing this stuff in an OSS repo.

File: src/client/unity.yml [hunk 1]
```diff
@@ -0,0 +1,280 @@
+openapi: "3.0.0"
+info:
+  title: Unity API Service
+  version: 0.1.0
+servers:
+  - url: /unity/v2 # to be determined
```
Comment (gunnaraasen) [inline src/client/unity.yml:6]: Maybe simply `/ui/v2`.

File: src/client/unity.yml [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+paths:
+  "/orgs/{orgID}/invites":
```
Comment (gunnaraasen) [inline src/client/unity.yml:8]: Should there also be a `get:` method to fetch the list of invites?

**Review by gunnaraasen — COMMENTED** [review]
Thanks @121watts for taking a first run at this spec. 

Since this will be an API primarily for UI use in InfluxDB Cloud, I agree with Randy's suggestion this spec should live in the quartz repo until we're ready for the UI API routes to be publicly consumable.

**Comment by 121watts** [comment]
@gunnaraasen before this code is live we can horse trade and decide where stuff should live.  for now, im going to keep it here as this is the most convenient place for the `ui` team to consume it.

File: src/client/unity.yml (hunk 1 above)
Comment (randycoulman) [reply src/client/unity.yml:6]: When this moves to Quartz, it will be `/api/v2private`.

File: src/client/unity.yml (hunk 2 above)
Comment (randycoulman) [reply src/client/unity.yml:8]: The list of invites comes back when fetching the user list: you get back both users and invites in that response.

**Pull request merged**